src/quo/components/refreshable_scroll_view/view.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo.components.refreshable-scroll-view.view
```

Comment:
Quo components should really be only things related to the design system file.

Imo this seems better to have in status-im.common... ns